Content.Server/Medical/SuitSensors/SuitSensorSystem.cs

```diff
@@ -205,7 +212,13 @@ private void OnVerb(EntityUid uid, SuitSensorComponent component, GetVerbsEvent<
             return;
 
         // standard interaction checks
-        if (!args.CanAccess || !args.CanInteract || args.Hands == null)
+        if (!args.CanInteract || args.Hands == null)
+            return;
+
+        if (!_interactionSystem.InRangeUnobstructed(args.User, args.Target))
+            return;
+
+        if (component.User != null && args.User != component.User && _actionBlocker.CanInteract(component.User.Value, null))
```

Comment:
Please leave a comment indicating that this is to check if the target wearing the jumpsuit is not incapacitated